README.md

```diff
@@ -355,17 +355,6 @@ After this, re-build and restart using `yarn build` and `yarn start` as document
 
 **NOTE**: At this time, it's not possible to sign in using this approach.
 
-### Working with UX Mocks
-
-When implementing user interfaces you will need to refer to the [Sketch](https://www.sketchapp.com/) mocks that are located in the [assets](https://github.com/mozilla/addons-frontend/tree/master/assets) directory. You will need a license to run Sketch and you also need to install some fonts (which are free). Install [Fira Sans](https://www.fontsquirrel.com/fonts/fira-sans), [Open Sans](https://www.fontsquirrel.com/fonts/open-sans), [Fira Mono](https://www.fontsquirrel.com/fonts/fira-mono) and [Chivo](https://www.fontsquirrel.com/fonts/chivo).
```

Comment:
Let's keep this section but link to https://mozilla.invisionapp.com/share/YCCYEOFRH#/screens (I don't know how to link to the directory of screens).

We still need to know where the UX mocks are and it's especially useful for QA -- they had been using Sketch files, I think. The Invision link above appears to be public which is what we want.

Reply:
ah right, that makes sense